src/main/scala/slamdata/SbtSlamData.scala

```diff
@@ -273,6 +287,70 @@ object SbtSlamData extends AutoPlugin {
   private def transferScripts(baseDir: File, srcs: String*) =
     srcs.foreach(src => transfer(src, baseDir / "scripts" / src, Set(OWNER_EXECUTE)))
 
+  private def slamdataVersionCheck = Def.task {
+    import scala.sys.process._
+
+    val logger: Logger = streams.value.log
+    def relog(line: String): Unit = {
+      val level = """\[([^]]*)\].*""".r
+      line match {
+        case level("debug") => logger.debug(line)
+        case level("info") => logger.info(line)
+        case level("warn") => logger.warn(line)
+        case level("error") => logger.error(line)
+        case level("success") => logger.success(line)
+        case _ => logger.info(line)
+      }
+    }
+    val logToSbt = ProcessLogger(relog _)
+
+    val getCsbt = "wget -O csbt https://github.com/coursier/sbt-extras/raw/master/sbt"
+    val csbtUpdate: Seq[String] = Seq(
+      "/usr/bin/env",
+      "bash",
+      "./csbt",
+      """set versionReconciliation += "com.slamdata" % "*" % "strict"""",
+      "update")
+    val sbtUpdate = "./sbt update"
+
+    logger.info("running slamdata version check")
+
+    val pluginFile = scala.reflect.io.File("project/coursier.sbt")
+    val pluginCmd = """addSbtPlugin("io.get-coursier" % "sbt-coursier" % "2.0.0-RC5-2")"""
+
+    val check = Try {
+      def ensureCoursierSbt = {
+        if (!file("csbt").exists) {
+          logger.info("downloading coursier sbt-extras to csbt")
+          logger.debug(getCsbt)
+          if ((getCsbt ! logToSbt) != 0) {
+            throw new IllegalStateException("unable to download coursier sbt-extras")
+          }
+        }
+      }
+
+      ensureCoursierSbt
```

Comment:
Style note: `ensureCoursierSbt()`. Actually I would just straight inline the function, since you're only using it here.